roles/v2/pool/src/lib/mining_pool/mod.rs

```diff
@@ -233,6 +236,7 @@ impl Pool {
             config.listen_address
         );
         while let Ok((stream, _)) = listener.accept().await {
+            let address = stream.peer_addr().unwrap();
```

Comment:
is it safe to unwrap this?

Reply:
yeah - it looks safe. In the example code they are using `?` - I followed the call down to the bottom and it looks safe to me.